server/src/assets/admin.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<link rel="icon" type="image/png" id=favIcon href="favicon.jpg" />
+
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.2.1/dist/css/bootstrap.min.css" />
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap4-toggle@3.6.1/css/bootstrap4-toggle.min.css" />
+
+	<script src="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/bootstrap4-toggle@3.6.1/js/bootstrap4-toggle.min.js"></script>
+
+	<link rel="stylesheet" href="style.css" />
+
+	<script src="script.js"></script>
+
+	<title>SWTbahn Admin Client</title>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+</head>
+
+<body>
+	<div class="row">
+		<div class="col-sm-6">
+			<div id="swtLogo"></div>
+		</div>
+		<div class="col-sm-6">
+			<div id="userLogo">Admin</div>
+		</div>
+	</div>
+	<hr />
+
+	<div class="card-deck">
+		
+		<div class="card" style="width: 22rem;">
+			<div class="card-body">
+				<h3 class="card-title"> System</h3>
+				<div class="form-group">
+					<button id=pingButton type="button" class="btn btn-outline-primary btn-block">Ping SWTbahn Server</button>
+					<br />
+					<div class="alert alert-info" role="alert">
+						<span id=pingResponse></span>
+					</div>
+				</div>
+				<br />
+				<div class="form-group">
+					<div style="display: flex; justify-content: space-between;">
+						<button id=startupButton type="button" class="btn btn-outline-primary" style="width: 49%;">Startup</button>
+						<button id=shutdownButton type="button" class="btn btn-outline-primary" style="width: 49%;">Shutdown</button>
+					</div>
+					<br />
+					<div class="alert alert-info" role="alert">
+						<span id=startupShutdownResponse></span>
+					</div>
+				</div>
+			</div>
+		</div>
+		
+		<div class="card" style="width: 22rem;">
+			<div class="card-body">
+				
+				<h3 class="card-title">  Verification</h3>
+				<div class="form-group">
+					<label style="color:#007bff">Verification for Train Engines </label>
+					<input type="checkbox" id="verificationToggle" data-toggle="toggle" data-size="xs">
+					<br />
+					<div class="alert alert-info" role="alert">
+						<span id=verificationSettingResponse></span>
+					</div>
```

Comment:
Move to `client.html` with plain toggle

Reply:
This will be done in second round of adjustments - will first test all changes with the admin.html, then move to client.html. Then test again.